### PR TITLE
fix: change name of the chartmuseum ingress resource

### LINKED ISSUE #7340: jx get urls doesn't show chart museum..
### Summary 

`jx get urls` used to display chart museum url now it doesn't show it anymore, but there is an ingress exposed for it. It will be really useful to have it back. This is a regression. 

### Steps to reproduce the behavior
`jx get urls`

### Expected behavior

show chart museum url in the output

### Actual behavior

does it show it 

### Jx version

The output of `jx version` is:

```
salaboy> jx version
NAME               VERSION
jx                 2.1.62
Kubernetes cluster v1.16.9-gke.2
kubectl            v1.13.2
helm client        2.12.2
git                2.24.3 (Apple Git-128)
Operating System   Mac OS X 10.15.4 build 19E287
```

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

fixes: https://github.com/jenkins-x/jx/issues/7340

### CODE REVIEW

File: jxboot-resources/templates/700-chartmuseum-ing.yaml [hunk 1]
```diff
@@ -11,14 +11,14 @@ metadata:
...
-  name: chartmuseum
+  name: {{.Release.Name}}-chartmuseum
 spec:
   rules:
   - host: chartmuseum{{ .Values.cluster.namespaceSubDomain }}{{ .Values.cluster.domain }}
     http:
       paths:
       - backend:
-          serviceName: jenkins-x-chartmuseum
+          serviceName: {{.Release.Name}}-chartmuseum
```
Comment (cagiti) [inline jxboot-resources/templates/700-chartmuseum-ing.yaml:21]: Not sure whether `{{.Release.Name}}` resolves to the value here https://github.com/jenkins-x/jenkins-x-platform/blob/90d20e869ef18843f5db375f2971f199559066e4/jenkins-x-platform/templates/chartmuseum-link-service.yaml#L5

Comment (ankitm123) [reply jxboot-resources/templates/700-chartmuseum-ing.yaml:21]: https://github.com/jenkins-x/jenkins-x-boot-config/blob/master/jenkins-x.yml#L223-L235 -> It passes `jenkins-x`, so I used the {{.Release.Name}}. I saw this charts created during the `install-jenkins-x` step.
The chart museum charts actually use `{{.Release.Name}}`.

Comment (cagiti) [reply jxboot-resources/templates/700-chartmuseum-ing.yaml:21]: perfect 👍

**Pull request merged**